inc/helpers/namespace.php

```diff
@@ -200,6 +200,12 @@ function get_default_menu( $items = '' ) {
 				admin_url(),
```

Comment:
A good option could be use: https://developer.wordpress.org/reference/functions/self_admin_url/
But doesn't work as expected for our current scheme and logic we want.